uqcs/templates/form.mako

```diff
@@ -40,7 +40,7 @@
           type="text"
           class="form-control"
           id="fnameInput"
-          placeholder="John"
+          placeholder=""
```

Comment:
Why are the placeholders removed? Please just replace with an androgynous first name + Smith.

Reply:
I didn't want to use placeholders for both sample inputs and help text but that's not really an issue. I'll add some names.